example_9/package.xml

```diff
@@ -23,8 +23,10 @@
 
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>forward_command_controller</exec_depend>
-  <exec_depend>gazebo_ros</exec_depend>
-  <exec_depend>gazebo_ros2_control</exec_depend>
+  <exec_depend condition="$GZ_VERSION == ''">ros_gz_sim</exec_depend>
+  <exec_depend condition="$GZ_VERSION == 'harmonic'">ros_gz_sim</exec_depend>
+  <exec_depend condition="$GZ_VERSION == 'garden'">ros_gz_sim</exec_depend>
```

Comment:
I just copied it over from [gz_ros2_control](https://github.com/ros-controls/gz_ros2_control/blob/master/gz_ros2_control_demos/package.xml) and deleted the fortress branch because I don't want to backport this to humble. Now it doesn't make sense, that's right. fixed with [bcdfedc](https://github.com/ros-controls/ros2_control_demos/pull/482/commits/bcdfedcbbe19550562bec133157d3bdd784981d5)